helpers/remote.py

```diff
@@ -259,4 +291,13 @@ def connect():
     for node in [client, server, tempesta, host]:
         node.mkdir(node.workdir)
 
+def wait_available():
+    global client
+    global server
+    global tempesta
+
+    client.wait_available()
```

Comment:
It not a first time I see this in PRs, you always make the same actions with all nodes without any loops. E.g. I would write:
```
for node in [client, server, tempesta]:
    node.wait_available()
```
Is there any practical reason to use non-loop approach?

And since there are more than one places, where all remote nodes are processed,  I would create special list with separate remote nodes.

Reply:
As for me, it's simpler to read. But may be i'm wrong